common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -351,4 +356,38 @@ public static String sanitizeNullAndLowercaseAndTrim(final String input) {
 
         return outValue;
     }
+
+    /**
+     * encode string with url form encoding. Space will be +.
```

Comment:
```suggestion
     * Encode string with url form encoding. Space will be +.
```

Reply:
"Space will be plus dot" ~ What does this mean?

Reply:
@shahzaibj I think it meant "space will be replaced by '+'". dot is just a fullstop.